bin/rad-issues

```diff
@@ -2,17 +2,23 @@
 
 (load! (find-module-file! "prelude.rad"))
 (load! (find-module-file! "monadic/issues.rad"))
+(load! (find-module-file! "monadic/project.rad"))
 (load! (find-module-file! "prelude/io-utils.rad"))
 (load! (find-module-file! "prelude/cmd-parsing.rad"))
 
 (import prelude/validation :as 'validation)
 
-(def base-name "http://machines.radicle.xyz/chains/")
+(def make-chain-ref
+  (fn [chain]
+    (ref (chain/load-chain! chain))))
+
+(def base-name
+  (fn []
+    (lookup :id (first-rsm-of-type! (make-chain-ref (get-project-url!)) :rad-issues))))
```

Comment:
This line is duplicated in `bin/rad-diff` and is probably something that is used frequently. How about `monadic/project` exporting a helper function for this?